network/fuzz_test.go

```diff
@@ -0,0 +1,19 @@
+package network // import "collectd.org/network"
+
+import (
+	"io/ioutil"
+	"testing"
+)
+
+func TestFuzz(t *testing.T) {
+	data, err := ioutil.ReadFile("testdata/packet2.bin")
+	if err != nil {
+		panic(err)
+	}
+
+	got := Fuzz(data)
```

Comment:
Doesn't this file also need the `// +build gofuzz` marker? I'd expect *Fuzz()* to be undefined when running `go test` …